homeworks/07-mars-viewer/src/components/images/images.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import {useSelector} from "react-redux";
+import {selectImagesBySol, selectIsLoading} from "../../store/store";
+import Image from "../image/image";
+
+// styles
+import s from "./images.module.scss";
+
+const Images: React.FC = () => {
+	const images = useSelector(selectImagesBySol);
+	const isLoading = useSelector(selectIsLoading);
+
+	return (
+		<>
+			{
+				isLoading
+					? <div className={s.text}>Loading...</div>
```

Comment:
better not to write nested ternary, eslint even has rule for that, could be converted in couple of `if`-s